### ISSUE TITLE
serve --prod fails 

### ISSUE THREAD

**Issue by Abenezer** [snics/ng2-avatar#2]
after we import  to app module 
`import { AvatarComponent } from 'ng2-avatar'
where shoud we put it 
is it in  declaration, import ... 
if put under declaration, ng serve works but ng serve --prod fails

**Comment by snics** [comment]
Hi Abenezer,
I fixed the error with the new version v2.0.6 for the new version install ```npm i ng2-avatar@2.0.6 --save``` and modify the import statement:
```
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { FormsModule } from '@angular/forms';
import { HttpModule } from '@angular/http';
import { AppComponent } from './app.component';
 
import { AvatarModule } from "ng2-avatar";
 
@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule,
    FormsModule,
    HttpModule,
    AvatarModule.forRoot()
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

**Comment by Abenezer** [comment]
yes, its working now, thanks for quick fix
